randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -1171,7 +1231,16 @@ Extra - asset_id: collision_camera_044
   * Layers: default
   * Open Passage to Metroid Caverns Save Station/Dock to Metroid Caverns Hub
   > Tunnel to Metroid Caverns Save Station
-      Lay Any Bomb
+      All of the following:
+          # Destroy Blocks
+          Lay Any Bomb
+          Any of the following:
+              # Get up
+              Single-wall Wall Jump (Beginner) or Super Jump (Intermediate) or Unmorph Extend (Beginner) or Climb Rooms Vertically (High Jump)
```

Comment:
This grouping is actually impossible with Bombs. Breaking the block with Bombs and just High Jump or Space Jump needs more things. I'd separate this out or not use the template here.